### ISSUE TITLE
NVDA is not reading "Change Keybinding" button name

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52027]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Deatils:**
Screenreader,NVDA[2018.2],MAS1.3.1

**Repro Steps:**
1.Launch VS Code while NVDA is ON.
2.Navigate to Settings icon in Activity Bar->Press Enter->Select "Keyboard Shortcuts" from dropdown.
3.Navigate to "Change Keybinding" button.
4.Observed that screenreader is not reading the name of button.

**Actual:**
NVDA is not reading the "Change Keybinding" button name.This button is actionable and NVDA is not reading it.

**Expected:**
NVDA should reads the button name as "Change Keybinding" button.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](url)

**User Impact:**
Screenreader is not reading button name because of that, user will bot understand where the keyboard focus is and what action user have to perform.
User will not take appropriate action on the button becuase narrator is not announcing it's name.

**MAS Reference:**
MAS1.3.1- [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](url)

**Attachment For Reference:**
[[Accessibility]A11y_VSCode_KeyboardShortcuts_ScreenReader.pptx](https://github.com/Microsoft/vscode/files/2105927/Accessibility.A11y_VSCode_KeyboardShortcuts_ScreenReader.pptx)
Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
I can repro this, and I feel like this is law hanging fruit, let's leave it for July

**Comment by sandy081** [comment]
This is an action item with title, not sure why the screen reader is not reading the title.

@bpasero any idea?

**Comment by bpasero** [comment]
@sandy081 @isidorn this fails for any action within tree/list, check with @joaomoreno

**Comment by isidorn** [comment]
Also assigning to @joaomoreno and me so we investigate

**Comment by joaomoreno** [comment]
Duplicate of #52682
